src/main/java/com/example/book/store/model/Book.java

```diff
@@ -0,0 +1,78 @@
+package com.example.book.store.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+
+@Entity
+@Table(name = "books")
+public class Book {
```

Comment:
Use the library Lombok
then you will be able to use the annotations like @Data, @Setter, @Getter